2023-09-07 22:50:36,323 - root - INFO - This is a log message
2023-09-07 22:50:36,358 - root - INFO - load_config.py start
2023-09-07 22:50:40,861 - root - INFO - 本机与远程主机连通性测试失败
2023-09-07 22:50:40,861 - root - INFO - 获取系统版本为 openEuler
2023-09-07 22:50:40,861 - root - INFO - sysctl_res_file_name = 'sysctl@openEuler.txt',ulimit_res_file_name = 'ulimit@openEuler.txt',save_difflib_res = 'differ-2023-09-07-22-50-36.txt',save_statistical_res = 'statistical-2023-09-07-22-50-36.txt' 
2023-09-07 22:50:40,872 - root - INFO - run command 'sysctl -a'
2023-09-07 22:50:40,872 - root - INFO - run_sysctl_command_and_save_result successfully, save in './data/sysctl@openEuler.txt'
2023-09-07 22:50:40,873 - root - INFO - run command 'ulimit -a'
2023-09-07 22:50:40,873 - root - INFO - run_ulimit_command_and_save_result successfully, save in './data/ulimit@openEuler.txt'
2023-09-07 22:50:48,678 - root - INFO - compare file1 = './data/sysctl@openEuler.txt', file2 = './data/sysctl@CentOSStream.txt'
2023-09-07 22:50:48,679 - root - INFO - text1 in './data/sysctl@openEuler.txt', text2 in './data/sysctl@CentOSStream.txt'
2023-09-07 22:50:48,679 - root - INFO - compare file1 and file2 by difflib successfully
